products/apache-cassandra.md

```diff
@@ -14,46 +14,48 @@ auto:
 
 releases:
 -   releaseCycle: "4.1"
-    eol: 2025-05-01
+    eol: 2025-07-15
     support: true
     releaseDate: 2022-12-13
     latest: "4.1.2"
     latestReleaseDate: 2023-05-29
 -   releaseCycle: "4.0"
-    eol: 2024-05-01
+    eol: 2024-07-15
     support: true
     releaseDate: 2021-07-26
     latest: "4.0.10"
     latestReleaseDate: 2023-05-29
 -   releaseCycle: "3.11"
-    eol: 2023-05-01
+    eol: 2023-12-01
```

Comment:
oops. this should have been `eol: 2023-12-15` (to be the same as the next one below for 3.0)